manifests/charts/gateway/values.yaml

```diff
@@ -96,5 +96,32 @@ imagePullPolicy: ""
 
 imagePullSecrets: []
 
-podDisruptionBudget:
-  minAvailable: 1
+# This value is used to configure a Kubernetes PodDisruptionBudget for the gateway.
+#
+# By default, the `podDisruptionBudget` is disabled (set to `{}`),
```

Comment:
That's a nice summary of the PDB doc - I was looking more for 'the value is a fragment of [link to PDB doc]' and 
'in a test/small/dev environment it is safe to leave it off, in production we recommend setting it to X'. 

I don't actually know what is the right X - some examples would help if someone has run this in prod.
Does it depend on cluster size ( a large ingress with ~10 replicas would use 1 as a value or 4 ? ). 
Do we recommend or not unhealtyPodEvictionPolicy for ingress ? 

That's the real problem - we need to give some operational advice to users. I'm a developer not an SRE - so 
people who run this in prod are much more qualified.

We probably need to have this on the wiki or doc site too - but most users just look at the helm params,
AFAIK the docs show up.